mission_systems/shooter/nodes/ShooterControl.py

```diff
@@ -140,28 +158,42 @@ def stop_actions(self):
           self.stop = True
         else:
           self.stop = False
+          self.motor_controller.setMotor1(self.motor1_stop)
+          self.motor_controller.setMotor2( self.motor2_stop)       
           self.motor1_stop = 0
-          self.motor2_stop = 0       
+          self.motor2_stop = 0
+
     def cancel_callback(self, req):
+        self.manual_used = True
         self.motor1_stop = 0
         self.motor2_stop = 0
         self.stop_actions()
         rospy.loginfo("canceled")
         return TriggerResponse(success=True)
+
     def manual_callback(self, req):
+        self.manual_used = True
         self.motor1_stop = -req.feeder
         self.motor2_stop = req.shooter
         self.stop_actions()
         res = ShooterManualResponse()
         res.success = True
         return res
 
+    def reset_callback(self,data):
+        self.stop_actions()
+        self.manual_used = False
+        return TriggerResponse(success=True)
+
     def is_joy_shoot(self,data): #Shoot if right trigger is held down
```

Comment:
The pep train has no brakes:
Spaces after function arguments (E241)
Those comments should be docstrings